apps/web/src/actions/auth.ts

```diff
@@ -372,3 +380,35 @@ export async function signOutDemoAction() {
     cookieStore.delete("demo_mode");
     redirect("/");
 }
+
+export async function updateReportConfigSettings(data: z.infer<typeof reportSettingsSchema>, userId: string | null) {
+    let id = userId;
+    if (!id) {
+        const { user } = await getActionSession();
+
+        if (!user) {
+            throw new UserNotLoggedInError();
```

Comment:
Bitte in Server Actions keine throws. Da an den anderen Actions orientieren.
Die Errors werden in Production nicht ausgeführt, bzw. Nicht an den Client übergeben.